src/Esfa.Roaao.Xslx.IntegrationTests/ProdAppSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Sfa.Das.Sas.Indexer.ApplicationServices.Shared.Settings;
+using Sfa.Das.Sas.Indexer.Core.Services;
+
+namespace Esfa.Roaao.Xslx.IntegrationTests
+{
+    internal class ProdAppSettings : AppServiceSettings, IAppServiceSettings
+    {
+        public ProdAppSettings(IProvideSettings settingsProvider) : base(settingsProvider)
+        {
+        }
+
+        public new string VstsAssessmentOrgsUrl => base.VstsAssessmentOrgsUrl.Replace("local", "prod");
```

Comment:
we should be storing the url format with a {0} here so you can decide with one to use without having to do a replace like this. but we can do that later ;)